app/src/org/commcare/android/database/connect/models/PersonalIdSessionData.kt

```diff
@@ -41,6 +41,8 @@ data class PersonalIdSessionData(
     var otpFallback: Boolean = false,
     // the number of times the user hit the "resend code" button
     var otpReattempts: Int = 0,
```

Comment:
do we still need this ? (given it can be derived from `otpAttempts`)

Reply:
I feel like we should so that we know to fallback to Twilio when there are currently "0" reattempts, which I think might be more idiomatic. But I'm open to changing this. What are you thinking? Use `otpAttempts` instead and fallback to Twilio when the current value of `otpAttempts` is "1"?

Reply:
Cool cool!